packages/flutter_tools/lib/src/base/user_messages.dart

```diff
@@ -69,13 +69,14 @@ class UserMessages {
   String androidBadSdkDir(String envKey, String homeDir) =>
       '$envKey = $homeDir\n'
       'but Android SDK not found at this location.';
-  String androidMissingSdkInstructions(String envKey, Platform platform) =>
+  String androidMissingSdkInstructions(Platform platform) =>
       'Unable to locate Android SDK.\n'
       'Install Android Studio from: https://developer.android.com/studio/index.html\n'
       'On first launch it will assist you in installing the Android SDK components.\n'
       '(or visit ${_androidSdkInstallUrl(platform)} for detailed instructions).\n'
-      'If the Android SDK has been installed to a custom location, set $envKey to that location.\n'
-      'You may also want to add it to your PATH environment variable.\n';
+      'If the Android SDK has been installed to a custom location, please use\n'
+      '`flutter config --android-studio-dir` or\n'
+      '`flutter config --android-sdk` to update to that location.\n';
```

Comment:
I don't think `flutter config --android-sdk` helps finding the Android SDK, at least based on how the tool is written now.  Just suggesting `flutter config --android-sdk` is probably enough. 

https://github.com/flutter/flutter/blob/d3515f5fb6342e76cb8b99b9874605179a0d0592/packages/flutter_tools/lib/src/android/android_sdk.dart#L71

Reply:
Thanks for pointing that out. I've updated it by removing line 78. Please let me know if it looks good!